sourcegraph/about
# Sourcegraph 4 landing page updates for launch day

This closes #5687 with updates to the 4.0 landing page for launch day.

### Changelog
- adds react-twitch-embed package instead of reinventing the wheel. It's a nice robust package that supports tree shaking and great options for interactive/non interactive twitch streams.
- removes timer countdown, switches around content, and changes copy throughout the page
- adds new social image meta for launch

#### TODO:
- [x] add blog post image thumbnails; placeholders are there for now until they're ready
- [ ] update YouTube component with ID for highlights video 
- [x] update cta styles when CtaSection is modified to match new specs in #5739 

### Test

1. Ensure prettier has standardized the proposed changes.
2. Ensure twitch stream renders (tested earlier with Fabs/Olivia and the stream works)
3. Ensure youtube video plays
4. Ensure links go to their corresponding locations
